tendrl.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Similar, less serious note about naming here: would it make sense to rename it to `tendrl_server`?